### PR TITLE
Presentation proposal: Cypress component & e2e test

### PR DESCRIPTION
**Pull request by ghost**
# Presentation Proposal
A Cypress component & e2e test tutorial where we shortly explain: 
- What end-to-end testing is and why it’s important.
- How to set up a basic test suite against a react app. 
- E2E example test candidates:
- - Login test
- Component tests candidates:
- - Forms

## Members
- Agnes Petäjävaara <agnespet@kth.se> anorangesky(https://github.com/anorangesky)
- Adam Henriksson <adahen@kth.se> adamih(https://github.com/Adamih)

### CODE REVIEW

File: contributions/presentation/week6/agnespet-adahen/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+# Presentation Proposal
```
Comment (cesarsotovalero) [inline contributions/presentation/week6/agnespet-adahen/README.md:2]: What is the novelty of your presentation? Can you explicitly make a novelty statement?

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]
Please provide more details.

File: contributions/presentation/week6/agnespet-adahen/README.md (hunk 1 above)
Comment (ghost) [reply contributions/presentation/week6/agnespet-adahen/README.md:2]: I've updated the PR with better info

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
@anorangesky, very good. I'll merge!

**Pull request merged**